Introduce ContainerNodeInlines.h

rniwa opened the pull request:
#### 3601b946b5e4152961a86ce2c1d45212200151e1
<pre>
Introduce ContainerNodeInlines.h
<a href="https://bugs.webkit.org/show_bug.cgi?id=292369">https://bugs.webkit.org/show_bug.cgi?id=292369</a>

Reviewed by Chris Dumez.

This PR introduces ContainerNodeInlines.h and moves a few inline functions there.
This helps reducing the build time of WebKit.

* Source/WebCore/Headers.cmake:
* Source/WebCore/Modules/model-element/HTMLModelElement.cpp:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/accessibility/AXObjectCache.cpp:
* Source/WebCore/accessibility/AccessibilityImageMapLink.cpp:
* Source/WebCore/accessibility/AccessibilityListBoxOption.cpp:
* Source/WebCore/accessibility/AccessibilityObject.cpp:
* Source/WebCore/accessibility/AccessibilitySpinButton.cpp:
* Source/WebCore/accessibility/atspi/AccessibilityObjectDocumentAtspi.cpp:
* Source/WebCore/animation/DocumentTimeline.cpp:
* Source/WebCore/animation/ScrollTimeline.cpp:
* Source/WebCore/animation/TimelineRange.cpp:
* Source/WebCore/css/ComputedStyleExtractor.cpp:
* Source/WebCore/css/calc/CSSCalcTree+ContainerProgressEvaluator.cpp:
* Source/WebCore/dom/Attr.cpp:
* Source/WebCore/dom/ContainerNode.h:
(WebCore::ContainerNode::ContainerNode):
* Source/WebCore/dom/ContainerNodeInlines.h: Copied from Source/WebCore/rendering/RenderFrame.h.
(WebCore::ContainerNode::rootNode const):
(WebCore::ContainerNode::protectedRootNode const):
(WebCore::ContainerNode::renderer const):
(WebCore::ContainerNode::checkedRenderer const):
* Source/WebCore/dom/ContentVisibilityDocumentState.cpp:
* Source/WebCore/dom/Document.cpp:
* Source/WebCore/dom/DocumentFullscreen.cpp:
* Source/WebCore/dom/Element.cpp:
* Source/WebCore/dom/EventPath.cpp:
* Source/WebCore/dom/ImageOverlay.cpp:
* Source/WebCore/dom/LiveNodeListInlines.h:
* Source/WebCore/dom/Node.cpp:
* Source/WebCore/dom/Position.cpp:
* Source/WebCore/dom/Range.cpp:
* Source/WebCore/dom/ShadowRoot.cpp:
* Source/WebCore/dom/StaticRange.cpp:
* Source/WebCore/dom/TreeScope.cpp:
* Source/WebCore/dom/TreeScopeInlines.h:
(WebCore::ContainerNode::rootNode const): Deleted.
(WebCore::ContainerNode::protectedRootNode const): Deleted.
* Source/WebCore/dom/TreeScopeOrderedMap.cpp:
* Source/WebCore/dom/ViewTransition.cpp:
* Source/WebCore/dom/mac/ImageControlsMac.cpp:
* Source/WebCore/editing/AppendNodeCommand.cpp:
* Source/WebCore/editing/DeleteSelectionCommand.cpp:
* Source/WebCore/editing/Editor.cpp:
* Source/WebCore/editing/IndentOutdentCommand.cpp:
* Source/WebCore/editing/RemoveNodeCommand.cpp:
* Source/WebCore/editing/ReplaceSelectionCommand.cpp:
* Source/WebCore/editing/SplitTextNodeContainingElementCommand.cpp:
* Source/WebCore/editing/TypingCommand.cpp:
* Source/WebCore/editing/VisibleSelection.cpp:
* Source/WebCore/editing/cocoa/HTMLConverter.mm:
* Source/WebCore/editing/ios/EditorIOS.mm:
* Source/WebCore/editing/mac/EditorMac.mm:
* Source/WebCore/editing/markup.cpp:
* Source/WebCore/html/BaseDateAndTimeInputType.cpp:
* Source/WebCore/html/CheckboxInputType.cpp:
* Source/WebCore/html/FileInputType.cpp:
* Source/WebCore/html/HTMLAreaElement.cpp:
* Source/WebCore/html/HTMLCollectionInlines.h:
* Source/WebCore/html/HTMLDetailsElement.cpp:
* Source/WebCore/html/HTMLEmbedElement.cpp:
* Source/WebCore/html/HTMLImageElement.cpp:
* Source/WebCore/html/HTMLInputElement.cpp:
* Source/WebCore/html/HTMLMeterElement.cpp:
* Source/WebCore/html/HTMLPlugInElement.cpp:
* Source/WebCore/html/HTMLSelectElement.cpp:
* Source/WebCore/html/HTMLSelectElementWin.cpp:
* Source/WebCore/html/HTMLTableCellElement.cpp:
* Source/WebCore/html/PluginDocument.cpp:
* Source/WebCore/html/RadioInputType.cpp:
* Source/WebCore/html/RangeInputType.cpp:
* Source/WebCore/html/TextFieldInputType.cpp:
* Source/WebCore/html/ValidatedFormListedElement.cpp:
* Source/WebCore/html/canvas/CanvasRenderingContext2D.cpp:
* Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp:
* Source/WebCore/html/parser/HTMLConstructionSite.cpp:
* Source/WebCore/html/parser/HTMLConstructionSite.h:
* Source/WebCore/html/parser/HTMLTreeBuilder.cpp:
* Source/WebCore/html/track/VTTRegion.cpp:
* Source/WebCore/inspector/agents/InspectorCSSAgent.cpp:
* Source/WebCore/loader/FrameLoader.cpp:
* Source/WebCore/loader/SubframeLoader.cpp:
* Source/WebCore/mathml/MathMLElement.cpp:
* Source/WebCore/mathml/MathMLPresentationElement.cpp:
* Source/WebCore/page/Chrome.cpp:
* Source/WebCore/page/DOMSelection.cpp:
* Source/WebCore/page/ElementTargetingController.cpp:
* Source/WebCore/page/FocusController.cpp:
* Source/WebCore/page/LocalFrameView.cpp:
* Source/WebCore/page/Page.cpp:
* Source/WebCore/page/PageSerializer.cpp:
* Source/WebCore/page/PrintContext.cpp:
* Source/WebCore/page/Quirks.cpp:
* Source/WebCore/page/ScrollBehavior.cpp:
* Source/WebCore/page/SpatialNavigation.cpp:
* Source/WebCore/page/ios/EventHandlerIOS.mm:
* Source/WebCore/page/ios/FrameIOS.mm:
* Source/WebCore/page/mac/ImageOverlayControllerMac.mm:
* Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp:
* Source/WebCore/page/scrolling/ScrollAnchoringController.cpp:
* Source/WebCore/page/text-extraction/TextExtraction.cpp:
* Source/WebCore/platform/win/PasteboardWin.cpp:
* Source/WebCore/rendering/BackgroundPainter.cpp:
* Source/WebCore/rendering/HitTestResult.cpp:
* Source/WebCore/rendering/InlineBoxPainter.cpp:
* Source/WebCore/rendering/PositionedLayoutConstraints.cpp:
* Source/WebCore/rendering/RenderAttachment.cpp:
* Source/WebCore/rendering/RenderCounter.cpp:
* Source/WebCore/rendering/RenderElement.h:
(WebCore::ContainerNode::renderer const): Deleted.
(WebCore::ContainerNode::checkedRenderer const): Deleted.
* Source/WebCore/rendering/RenderFrame.h:
* Source/WebCore/rendering/RenderMedia.h:
* Source/WebCore/rendering/RenderScrollbar.cpp:
* Source/WebCore/rendering/RenderTextControl.cpp:
* Source/WebCore/rendering/RenderTextControlMultiLine.h:
* Source/WebCore/rendering/style/StyleCachedImage.cpp:
* Source/WebCore/rendering/svg/RenderSVGGradientStop.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceClipper.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourcePattern.cpp:
* Source/WebCore/rendering/svg/RenderSVGTextPath.cpp:
* Source/WebCore/rendering/svg/SVGPathData.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGModelObject.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResource.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceMasker.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourcePattern.cpp:
* Source/WebCore/rendering/updating/RenderTreePosition.cpp:
* Source/WebCore/rendering/updating/RenderTreeUpdater.cpp:
* Source/WebCore/style/AnchorPositionEvaluator.cpp:
* Source/WebCore/style/StyleAdjuster.cpp:
* Source/WebCore/style/StyleScope.cpp:
* Source/WebCore/style/values/primitives/StyleLengthResolution.cpp:
* Source/WebCore/svg/SVGAnimateMotionElement.cpp:
* Source/WebCore/svg/SVGElement.cpp:
* Source/WebCore/svg/SVGFEDiffuseLightingElement.cpp:
* Source/WebCore/svg/SVGFEImageElement.cpp:
* Source/WebCore/svg/SVGFESpecularLightingElement.cpp:
* Source/WebCore/svg/SVGFilterPrimitiveStandardAttributes.cpp:
* Source/WebCore/svg/SVGForeignObjectElement.cpp:
* Source/WebCore/svg/SVGImageElement.cpp:
* Source/WebCore/svg/SVGLocatable.cpp:
* Source/WebCore/svg/SVGPathElement.cpp:
* Source/WebCore/svg/SVGPatternElement.cpp:
* Source/WebCore/svg/SVGRadialGradientElement.cpp:
* Source/WebCore/svg/SVGSVGElement.cpp:
* Source/WebCore/svg/SVGStopElement.cpp:
* Source/WebCore/svg/SVGTextContentElement.cpp:
* Source/WebCore/svg/SVGUseElement.cpp:
* Source/WebCore/svg/graphics/SVGImage.cpp:
* Source/WebCore/svg/properties/SVGAnimationAdditiveValueFunctionImpl.cpp:
* Source/WebCore/testing/Internals.cpp:
* Source/WebKit/WebProcess/Automation/WebAutomationSessionProxy.cpp:
* Source/WebKit/WebProcess/FullScreen/WebFullScreenManager.cpp:
* Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm:
* Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm:
* Source/WebKit/WebProcess/Plugins/PluginView.cpp:
* Source/WebKitLegacy/mac/DOM/DOM.mm:
* Source/WebKitLegacy/mac/DOM/DOMHTML.mm:
* Source/WebKitLegacy/mac/DOM/DOMHTMLInputElement.mm:
* Source/WebKitLegacy/mac/DOM/DOMHTMLObjectElement.mm:
* Source/WebKitLegacy/mac/DOM/DOMUIKitExtensions.mm:
* Source/WebKitLegacy/mac/Misc/WebNSPasteboardExtras.mm:
* Source/WebKitLegacy/mac/WebCoreSupport/WebFrameLoaderClient.mm:
* Source/WebKitLegacy/mac/WebView/WebFullScreenController.mm:
* Source/WebKitLegacy/mac/WebView/WebHTMLView.mm:

Canonical link: <a href="https://commits.webkit.org/294407@main">https://commits.webkit.org/294407@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f302d595cee270acae1d7695c684071e5d30987b

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/101695 "4 style errors") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/21363 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/11679 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/106853 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/52329 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/103735 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/21671 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/29864 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/77441 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/34465 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/104702 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/16742 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/91836 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/57778 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/16570 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/9853 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/51679 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/86432 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/9930 "Passed tests") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/109207 "Build is in progress. Recent messages:Printed configuration; Skipping applying patch since patch_id isn't provided; Checked out pull request; Compiled WebKit") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/28828 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/21231 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/86416 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/29189 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/88037 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/85980 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/30736 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/8453 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/22991 "Built successfully") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/16549 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/28756 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/34046 "Built successfully") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/28567 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/31890 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/30126 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

cdumez commented on Source/WebCore/dom/TreeScopeInlines.h:
```diff
@@ -30,18 +30,18 @@
     return ensureIdTargetObserverRegistry();
 }
 
+inline ContainerNode& TreeScope::rootNode() const
```
Not sure why this one moved.

rniwa replied:
Just a cleanup. It's nice to have each function in its respective class's `Inlines.h`.

cdumez replied:
My point is that you moved it *within* the same `Inlines.h` file it seems, so we could have reduced the diff by keeping it where it was.

rniwa replied:
Oh weird. Will revert.

rniwa committed:
Introduce ContainerNodeInlines.h
https://bugs.webkit.org/show_bug.cgi?id=292369

Reviewed by Chris Dumez.

This PR introduces ContainerNodeInlines.h and moves a few inline functions there.
This helps reducing the build time of WebKit.

* Source/WebCore/Headers.cmake:
* Source/WebCore/Modules/model-element/HTMLModelElement.cpp:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/accessibility/AXObjectCache.cpp:
* Source/WebCore/accessibility/AccessibilityImageMapLink.cpp:
* Source/WebCore/accessibility/AccessibilityListBoxOption.cpp:
* Source/WebCore/accessibility/AccessibilityObject.cpp:
* Source/WebCore/accessibility/AccessibilitySpinButton.cpp:
* Source/WebCore/accessibility/atspi/AccessibilityObjectDocumentAtspi.cpp:
* Source/WebCore/animation/DocumentTimeline.cpp:
* Source/WebCore/animation/ScrollTimeline.cpp:
* Source/WebCore/animation/TimelineRange.cpp:
* Source/WebCore/css/ComputedStyleExtractor.cpp:
* Source/WebCore/css/calc/CSSCalcTree+ContainerProgressEvaluator.cpp:
* Source/WebCore/dom/Attr.cpp:
* Source/WebCore/dom/ContainerNode.h:
(WebCore::ContainerNode::ContainerNode):
* Source/WebCore/dom/ContainerNodeInlines.h: Copied from Source/WebCore/rendering/RenderFrame.h.
(WebCore::ContainerNode::rootNode const):
(WebCore::ContainerNode::protectedRootNode const):
(WebCore::ContainerNode::renderer const):
(WebCore::ContainerNode::checkedRenderer const):
* Source/WebCore/dom/ContentVisibilityDocumentState.cpp:
* Source/WebCore/dom/Document.cpp:
* Source/WebCore/dom/DocumentFullscreen.cpp:
* Source/WebCore/dom/Element.cpp:
* Source/WebCore/dom/EventPath.cpp:
* Source/WebCore/dom/ImageOverlay.cpp:
* Source/WebCore/dom/LiveNodeListInlines.h:
* Source/WebCore/dom/Node.cpp:
* Source/WebCore/dom/Position.cpp:
* Source/WebCore/dom/Range.cpp:
* Source/WebCore/dom/ShadowRoot.cpp:
* Source/WebCore/dom/StaticRange.cpp:
* Source/WebCore/dom/TreeScope.cpp:
* Source/WebCore/dom/TreeScopeInlines.h:
(WebCore::ContainerNode::rootNode const): Deleted.
(WebCore::ContainerNode::protectedRootNode const): Deleted.
* Source/WebCore/dom/TreeScopeOrderedMap.cpp:
* Source/WebCore/dom/ViewTransition.cpp:
* Source/WebCore/dom/mac/ImageControlsMac.cpp:
* Source/WebCore/editing/AppendNodeCommand.cpp:
* Source/WebCore/editing/DeleteSelectionCommand.cpp:
* Source/WebCore/editing/Editor.cpp:
* Source/WebCore/editing/IndentOutdentCommand.cpp:
* Source/WebCore/editing/RemoveNodeCommand.cpp:
* Source/WebCore/editing/ReplaceSelectionCommand.cpp:
* Source/WebCore/editing/SplitTextNodeContainingElementCommand.cpp:
* Source/WebCore/editing/TypingCommand.cpp:
* Source/WebCore/editing/VisibleSelection.cpp:
* Source/WebCore/editing/cocoa/HTMLConverter.mm:
* Source/WebCore/editing/ios/EditorIOS.mm:
* Source/WebCore/editing/mac/EditorMac.mm:
* Source/WebCore/editing/markup.cpp:
* Source/WebCore/html/BaseDateAndTimeInputType.cpp:
* Source/WebCore/html/CheckboxInputType.cpp:
* Source/WebCore/html/FileInputType.cpp:
* Source/WebCore/html/HTMLAreaElement.cpp:
* Source/WebCore/html/HTMLCollectionInlines.h:
* Source/WebCore/html/HTMLDetailsElement.cpp:
* Source/WebCore/html/HTMLEmbedElement.cpp:
* Source/WebCore/html/HTMLImageElement.cpp:
* Source/WebCore/html/HTMLInputElement.cpp:
* Source/WebCore/html/HTMLMeterElement.cpp:
* Source/WebCore/html/HTMLPlugInElement.cpp:
* Source/WebCore/html/HTMLSelectElement.cpp:
* Source/WebCore/html/HTMLSelectElementWin.cpp:
* Source/WebCore/html/HTMLTableCellElement.cpp:
* Source/WebCore/html/PluginDocument.cpp:
* Source/WebCore/html/RadioInputType.cpp:
* Source/WebCore/html/RangeInputType.cpp:
* Source/WebCore/html/TextFieldInputType.cpp:
* Source/WebCore/html/ValidatedFormListedElement.cpp:
* Source/WebCore/html/canvas/CanvasRenderingContext2D.cpp:
* Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp:
* Source/WebCore/html/parser/HTMLConstructionSite.cpp:
* Source/WebCore/html/parser/HTMLConstructionSite.h:
* Source/WebCore/html/parser/HTMLTreeBuilder.cpp:
* Source/WebCore/html/track/VTTRegion.cpp:
* Source/WebCore/inspector/agents/InspectorCSSAgent.cpp:
* Source/WebCore/loader/FrameLoader.cpp:
* Source/WebCore/loader/SubframeLoader.cpp:
* Source/WebCore/mathml/MathMLElement.cpp:
* Source/WebCore/mathml/MathMLPresentationElement.cpp:
* Source/WebCore/page/Chrome.cpp:
* Source/WebCore/page/DOMSelection.cpp:
* Source/WebCore/page/ElementTargetingController.cpp:
* Source/WebCore/page/FocusController.cpp:
* Source/WebCore/page/LocalFrameView.cpp:
* Source/WebCore/page/Page.cpp:
* Source/WebCore/page/PageSerializer.cpp:
* Source/WebCore/page/PrintContext.cpp:
* Source/WebCore/page/Quirks.cpp:
* Source/WebCore/page/ScrollBehavior.cpp:
* Source/WebCore/page/SpatialNavigation.cpp:
* Source/WebCore/page/ios/EventHandlerIOS.mm:
* Source/WebCore/page/ios/FrameIOS.mm:
* Source/WebCore/page/mac/ImageOverlayControllerMac.mm:
* Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp:
* Source/WebCore/page/scrolling/ScrollAnchoringController.cpp:
* Source/WebCore/page/text-extraction/TextExtraction.cpp:
* Source/WebCore/platform/win/PasteboardWin.cpp:
* Source/WebCore/rendering/BackgroundPainter.cpp:
* Source/WebCore/rendering/HitTestResult.cpp:
* Source/WebCore/rendering/InlineBoxPainter.cpp:
* Source/WebCore/rendering/PositionedLayoutConstraints.cpp:
* Source/WebCore/rendering/RenderAttachment.cpp:
* Source/WebCore/rendering/RenderCounter.cpp:
* Source/WebCore/rendering/RenderElement.h:
(WebCore::ContainerNode::renderer const): Deleted.
(WebCore::ContainerNode::checkedRenderer const): Deleted.
* Source/WebCore/rendering/RenderFrame.h:
* Source/WebCore/rendering/RenderMedia.h:
* Source/WebCore/rendering/RenderScrollbar.cpp:
* Source/WebCore/rendering/RenderTextControl.cpp:
* Source/WebCore/rendering/RenderTextControlMultiLine.h:
* Source/WebCore/rendering/style/StyleCachedImage.cpp:
* Source/WebCore/rendering/svg/RenderSVGGradientStop.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourceClipper.cpp:
* Source/WebCore/rendering/svg/RenderSVGResourcePattern.cpp:
* Source/WebCore/rendering/svg/RenderSVGTextPath.cpp:
* Source/WebCore/rendering/svg/SVGPathData.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGModelObject.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResource.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceMasker.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourcePattern.cpp:
* Source/WebCore/rendering/updating/RenderTreePosition.cpp:
* Source/WebCore/rendering/updating/RenderTreeUpdater.cpp:
* Source/WebCore/style/AnchorPositionEvaluator.cpp:
* Source/WebCore/style/StyleAdjuster.cpp:
* Source/WebCore/style/StyleScope.cpp:
* Source/WebCore/style/values/primitives/StyleLengthResolution.cpp:
* Source/WebCore/svg/SVGAnimateMotionElement.cpp:
* Source/WebCore/svg/SVGElement.cpp:
* Source/WebCore/svg/SVGFEDiffuseLightingElement.cpp:
* Source/WebCore/svg/SVGFEImageElement.cpp:
* Source/WebCore/svg/SVGFESpecularLightingElement.cpp:
* Source/WebCore/svg/SVGFilterPrimitiveStandardAttributes.cpp:
* Source/WebCore/svg/SVGForeignObjectElement.cpp:
* Source/WebCore/svg/SVGImageElement.cpp:
* Source/WebCore/svg/SVGLocatable.cpp:
* Source/WebCore/svg/SVGPathElement.cpp:
* Source/WebCore/svg/SVGPatternElement.cpp:
* Source/WebCore/svg/SVGRadialGradientElement.cpp:
* Source/WebCore/svg/SVGSVGElement.cpp:
* Source/WebCore/svg/SVGStopElement.cpp:
* Source/WebCore/svg/SVGTextContentElement.cpp:
* Source/WebCore/svg/SVGUseElement.cpp:
* Source/WebCore/svg/graphics/SVGImage.cpp:
* Source/WebCore/svg/properties/SVGAnimationAdditiveValueFunctionImpl.cpp:
* Source/WebCore/testing/Internals.cpp:
* Source/WebKit/WebProcess/Automation/WebAutomationSessionProxy.cpp:
* Source/WebKit/WebProcess/FullScreen/WebFullScreenManager.cpp:
* Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm:
* Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm:
* Source/WebKit/WebProcess/Plugins/PluginView.cpp:
* Source/WebKitLegacy/mac/DOM/DOM.mm:
* Source/WebKitLegacy/mac/DOM/DOMHTML.mm:
* Source/WebKitLegacy/mac/DOM/DOMHTMLInputElement.mm:
* Source/WebKitLegacy/mac/DOM/DOMHTMLObjectElement.mm:
* Source/WebKitLegacy/mac/DOM/DOMUIKitExtensions.mm:
* Source/WebKitLegacy/mac/Misc/WebNSPasteboardExtras.mm:
* Source/WebKitLegacy/mac/WebCoreSupport/WebFrameLoaderClient.mm:
* Source/WebKitLegacy/mac/WebView/WebFullScreenController.mm:
* Source/WebKitLegacy/mac/WebView/WebHTMLView.mm:

Canonical link: https://commits.webkit.org/294407@main
